### PR TITLE
Hide comment field when no table comment is set

### PR DESCRIPTION
**Pull request by ibennetch**
On the table structure tab, this hide table comments if there is none. See https://github.com/phpmyadmin/phpmyadmin/issues/12961#issuecomment-281040584

Before submitting pull request, please check that every commit:

- [x] Has proper Signed-Off-By
- [x] Has commit message which describes it
- [x] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [ ] Any new functionality is covered by tests

### CODE REVIEW

File: templates/table/structure/display_table_stats.phtml [hunk 1]
```diff
@@ -1,9 +1,11 @@
 <div id="tablestatistics">
     <fieldset>
         <legend><?= __('Information'); ?></legend>
-        <p> <strong> <?= __('Table comments:') ?> </strong>
-            <?= isset($showtable['TABLE_COMMENT']) ? htmlspecialchars($showtable['TABLE_COMMENT']) : '';?>
-        </p>
+        <?php if(! empty($showtable['TABLE_COMMENT'])) { ?>
```
Comment (nijel) [inline templates/table/structure/display_table_stats.phtml]: Using `empty()` is probably not a good idea as `empty('0')` is true, this ti will not display such comment.

PS: Generally I tend to think that `empty()` should not be used at all in most situations.

Comment (ibennetch) [inline templates/table/structure/display_table_stats.phtml]: Ah, yes. I guess I should instead test for isset (which I think is always true here anyway) and if `== ''`

Comment (nijel) [inline templates/table/structure/display_table_stats.phtml]: Either use `=== ''` or `strlen(...) === 0`. I'm not sure if isset is necessary here...

**Comment by ibennetch** [comment]
I've updated this with the requested changes.

I'm starting to wonder if this should target master because we're trying to freeze changes to QA_4_7.

**Comment by nijel** [comment]
IMHO such minor changes are certainly okay (and this is anyway mostly bugfix).

**Pull request merged**